### DISCUSSION TITLE
Retro-compatibility

### DISCUSSION THREAD

**Discussion by AlexPerathoner** [AlexPerathoner/SlimHUD#122] [General]
If anyone can try it out, on one of the OSs that is still to check, it would be of great help!

# Running
| status  | version  |  problem |  
|:---:|:---:|:---:|
| 🟩   | 10.13  |  |  
| 🟨   | 10.14  |   to check |
| 🟨   | 10.15  |  to check |   
| 🟨   | 12.1  |   to check |
| 🟨   | 11.1  |   to check |
| 🟩     |  13.1 |   |

# Compiling 

- won't fix, only use latest version of Xcode and macOS to compile and run UI Tests

| status  | version  |  problem |  
|:---:|:---:|:---:|
| 🟥   | 10.15  |  Sparkle 2 not compatible |   
| 🟥   | 11.1  |  App is not agent but is using custom policy, which is not compatible  |
| 🟨    |  12.1 |  only works with one version of Xcode |

**Comment by kaydenanderson** [comment]
hey! i am running ventura 13.1 and would love to test anything that you may need. i've been following your project for a while now and got an email about a week ago that there have been updates - i got suuper excited :D
let me know if i can help in any way!

**Comment by AlexPerathoner** [comment]
Hi @kaydenanderson ! Thank you so much!
This new version will mostly contain fixes and updates on the repository itself, to make it more maintainable – but this also means that more features are coming soon!

Unfortunately (or should I say fortunately) Ventura is the only system which I'm sure SlimHUD currently runs on 😅 
But you can support the project by starring it on GitHub, if you haven't done so already :)

**Comment by kaydenanderson** [comment]
awesome! will do :)
edit: and thanks for your awesome work on this project!
